index.js

```diff
@@ -0,0 +1,239 @@
+    in_reply_to_status_id: tweet.id_str
+  });
+}
+async function doBalance(tweet, msg) {
+  try {
+    const balance = await lbry.getBalance(id(tweet.user.id_str), config.get("bot.requiredConfirms")); // Amount of confirms before we can use it.
+    const post = await T.post("statuses/update", {
+      in_reply_to_status_id: tweet.id_str,
+      status: `@${tweet.user.screen_name} You have ${balance} LBC.`
+    });
+    logger.info(
+      `Sent balance command to ${tweet.user.screen_name}, tweet id: ${
+        tweet.id_str
+      }`
+    );
+  } catch (e) {
+    logger.error(e);
+  }
+}
+async function doDeposit(tweet, msg) {
+  try {
+    const post = await T.post("statuses/update", {
+      status: `@${tweet.user.screen_name} Your deposit address is ${await getAddress(id(tweet.user.id_str))}.`,
+      in_reply_to_status_id: tweet.id_str
+    });
+    logger.info(
+      `Sent deposit address to ${tweet.user.screen_name}, tweet id: ${
+        tweet.id_str
+      }`
+    );
+  } catch (e) {
+    logger.error(e);
+  }
+}
+async function doWithdraw(tweet, msg) {
+  try {
+  if (msg.length < 4) return doHelp(tweet, msg);
+  let address = msg[2];
+  let amount = getValidatedAmount(msg[3]);
+  if (amount === null) {
+    return await T.post("statuses/update", {
+      status: `@${tweet.user.screen_name} I don´t know how to withdraw that many credits...`,
+      in_reply_to_status_id: tweet.id_str
+    });
+  }
+  let txId = await lbry.sendFrom(id(tweet.user.id_str), address, amount);
+  await T.post("statuses/update", {
+    status: `@${tweet.user.screen_name} You withdrew ${amount} LBC to ${address}. \n${txLink(txId)}`,
+    in_reply_to_status_id: tweet.id_str
+  });
+  logger.info(
+    `User ${
+      tweet.user.screen_name
+    } withdraw ${amount} LBC to ${address}, tweet id: ${tweet.id_str}`
+  );
+  } catch (e) {
+    logger.error(e);
+  }
+}
+async function doTip(tweet, msg) {
+  try {
+    if (msg.length < 3) {
+      return doHelp(tweet, msg);
+    }
+    const amount = getValidatedAmount(msg[3]);
+    if (amount === null) {
+      return await T.post("statuses/update", {
+        status: `@${tweet.user.screen_name} I don´t know how to tip that many credits...`,
+        in_reply_to_status_id: tweet.id_str
+      });
+    }
+    const userToTip = tweet.entities.user_mentions.find(u => `@${u.screen_name}` === msg[2]).id_str;
+    await getAddress(id(userToTip)) // Call this to ensure user has an account.
+    if (userToTip === null) {
+      return await T.post("statuses/update", {
+        status: `@${tweet.user.screen_name} I could not find that user...`,
+        in_reply_to_status_id: tweet.id_str
+      });
+    }
+    const balanceFromUser = await lbry.getBalance(id(tweet.user.id_str), config.get("bot.requiredConfirms"));
+    if (balanceFromUser < amount) {
+      return await T.post("statuses/update", {
+        status: `@${tweet.user.screen_name} You tried to tip, but you are missing ${amount-balanceFromUser} LBC.`,
```

Comment:
`You tried tipping more than you have! You are ${amount-balanceFromUser} LBC short.`

Reply:
Will fix!